common/scala/src/main/scala/whisk/utils/JsHelpers.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package whisk.utils
+
+import spray.json.JsObject
+import spray.json.JsValue
+
+object JsHelpers {
+    implicit class JsObjectHelper(js: JsObject) {
+        def getFieldPath(path: List[String]): Option[JsValue] = {
+            path match {
+                case Nil       => Option(js)
+                case p :: Nil  => js.fields.get(p)
+                case p :: tail => js.fields.get(p) match {
+                    case Some(o: JsObject) => o.getFieldPath(tail)
```

Comment:
Probably not terribly efficient given that every recursive call involves creating a new instance of the `implicit` class